basics/README.md

```diff
@@ -27,6 +27,7 @@ to learn more about whatever you want after studying it.
 19. [Exceptions](exceptions.md)
 20. [Classes](classes.md)
 21. [Docstrings](docstrings.md)
+22. [List Comprehension](list-comprehension.md)
```

Comment:
It would be good to have this immediately after loops. (You have to change the numbering.)